mlir/test/Conversion/RockToGPU/multiple_kernels.mlir

```diff
@@ -13,22 +13,22 @@
 
 module  {
   func.func @step1(%arg0: memref<1x512x16x3x3xf32>, %arg1: memref<512x1x16x32x32xf32>, %arg2: memref<512x1x512x30x30xf32>) attributes {kernel, arch = "amdgcn-amd-amdhsa:gfx906"} {
-    rock.conv2d(%arg0, %arg1, %arg2) features = dot {arch = "amdgcn-amd-amdhsa:gfx906", dilations = [1 : index, 1 : index], filter_layout = ["g", "k", "c", "y", "x"], input_layout = ["ni", "gi", "ci", "hi", "wi"], output_layout = ["no", "go", "ko", "ho", "wo"], padding = [0 : index, 0 : index, 0 : index, 0 : index], strides = [1 : index, 1 : index]} : memref<1x512x16x3x3xf32>, memref<512x1x16x32x32xf32>, memref<512x1x512x30x30xf32>
+    rock.conv(%arg0, %arg1, %arg2) features = dot {arch = "amdgcn-amd-amdhsa:gfx906", dilations = [1 : index, 1 : index], filter_layout = ["g", "k", "c", "y", "x"], input_layout = ["ni", "gi", "ci", "hi", "wi"], output_layout = ["no", "go", "ko", "ho", "wo"], padding = [0 : index, 0 : index, 0 : index, 0 : index], strides = [1 : index, 1 : index]} : memref<1x512x16x3x3xf32>, memref<512x1x16x32x32xf32>, memref<512x1x512x30x30xf32>
```

Comment:
Let's go ahead and output the quoted strings here? Just so we have canonical-form IR in our tests.

Reply:
Not sure what you mean by "output the quoted strings";  something like "ngchw" instead of the list of strings?